container_pipeline/lib/openshift.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import time
+
+from container_pipeline.lib import settings
+
+from .lib.command import run
+
+
+class OpenshiftError(Exception):
+    pass
+
+
+class Openshift(object):
```

Comment:
For all the `oc` commands we're using here, is it plausible to use [this Python API](https://github.com/openshift/openshift-restclient-python) instead?